app/dashboard/dashboard.tsx

```diff
@@ -73,11 +83,13 @@ export const Dashboard = ({ projects = [], config, user }: DashboardProps) => {
             <DropdownMenuTrigger asChild>
               <Button variant="raw" aria-label="Menu Button">
                 <Flex gap="1" align="center">
-                  {user?.image ? (
-                    <Avatar src={user?.image} alt="Open menu" />
-                  ) : (
-                    <UserIcon color="white" />
-                  )}
+                  <Avatar.Root>
+                    {user?.image && <Avatar.Image src={user?.image} />}
+                    <StyledFallback delayMs={500}>
+                      {(user?.username || "X").charAt(0).toLocaleUpperCase()}
```

Comment:
could use email as fallback for username if username is really optional and can't be enforced

Reply:
Done! Gets username, email and then does the x fallback